### PR TITLE
Add uncertainty to table values

### PR DESCRIPTION
**Pull request by mwoodbri**
Proof-of-concept, displaying the error values that we currently have.

- [ ] Add column that is currently possible but missing
- [ ] Decide on name and type of error field within `TableDefinition` schema
- [ ] Decide on whether error field should accept simple column name and/for formulae
- [ ] Decide whether to use use `<b-tooltip>` (or directive) rather than `<abbr>`
- [ ] Formatting of tooltip
- [ ] Address outstanding issues with error values (see mrc-2104)
- [ ] Get hold of error values for other columns

Screenshot from 2021-01-11 15-27-09

### CODE REVIEW

File: src/app/static/src/tests/components/figures/dynamicTable.test.ts [hunk 1]
```diff
@@ -152,4 +152,102 @@ describe("dynamic table", () => {
...
+                format: "0.0a"
+            }
+        ];
+        const wrapper = mount(dynamicTable, {
+            propsData: {data, config, settings}
+        });
+        const rows = wrapper.findAll("tbody tr");
+        expect(rows.at(0).find("td").text()).toBe("display name for none");
+        expect(rows.at(0).findAll("td").at(1).text()).toBe("0"); // cases averted
+        expect(rows.at(0).findAll("td").at(2).text()).toBe("n/a"); // costs per case averted
```
Comment (EmmaLRussell) [inline src/app/static/src/tests/components/figures/dynamicTable.test.ts:245]: This `n/a` is evaluated because the formula divides by 0 in the none case? Can we include tests for the tooltips in none case too?

**Review by EmmaLRussell — COMMENTED** [review]
This seems like a good approach to me, and both branches fine to merge in pending feedback.

Do we need to check with Tom and Ellie that tooltips are their preferred way of showing errors in table?

For cases averted, the errors are still rounding to the nearest 10 cases aren't they? We should check that's right, it does look a bit weird. Similarly for rounded prevalence values.

File: src/app/static/src/tests/components/figures/dynamicTable.test.ts (hunk 1 above)
Comment (mwoodbri) [reply src/app/static/src/tests/components/figures/dynamicTable.test.ts:246]: Very good point - we now check that tooltips aren't rendered for non-numeric fields. This required some re-jigging to separate cell evaluation from formatting but I think it's an improvement overall.

**Comment by mwoodbri** [comment]
Rendering of tooltips, error value integrity and issue of rounding to be discussed tomorrow

**Comment by mwoodbri** [comment]
Tooltip approach confirmed as suitable by Ellie after meeting on 2021-01-19

@EmmaLRussell Are you ok for me to merge this considering changes since last review?

**Comment by EmmaLRussell** [comment]
Yes, go ahead and merge both of these, got some conflicts to resolve on the mintr branch. 

> Tooltip approach confirmed as suitable by Ellie after meeting on 2021-01-19
> 
> @EmmaLRussell Are you ok for me to merge this considering changes since last review?
